Core/src/SplineInterpretation/SplineInterpretation.cpp

```diff
@@ -862,12 +862,12 @@ carve::geom::vector<3>  OpenInfraPlatform::Core::SplineInterpretation::SplineInt
 }
 
 carve::geom::vector<3> OpenInfraPlatform::Core::SplineInterpretation::SplineInterpretation::tangentVectorFromDirection(
-	double direction) const throw(...)
+	const double direction) const noexcept(true)
 {
 	return carve::geom::VECTOR(cos(direction*M_PI / 180), sin(direction*M_PI / 180), 0.0);

```

Comment:
```suggestion
	return carve::geom::VECTOR(cos(direction*M_PI / 180.), sin(direction*M_PI / 180.), 0.0);
```

How come that direction is in degrees? Naturally, in C, angles are in radians. Where does this come from?

Reply:
Done in commit 3ca85d7.
These angles are calculated in the function `angleOfVectors2D` where they were converted to degree manually.
During the development process, I was focused on testing with CAD drawings where usually degree is used. According to https://standards.buildingsmart.org, both degree and radiant is possible in an ifc file. At this point, I probably missed to doublecheck what is used in the ifc example files.
With the current changes, everything program internal works in radian and just the print output to the console window is in radian and degree (just in case that the value in degree might still be usefull in some further usage or human interpretation).

Reply:
Right, I understand. Rule of thumb:

- Internally, OIP works in meters and radians. 
- Whenever we print, we may decide and reformat to any other unit.